src/core/sys/posix/aio.d

```diff
@@ -63,6 +63,19 @@ version (CRuntime_Glibc)
         }
     }
 }
+else version (OSX)
```

Comment:
Other modules are using `Darwin`: https://github.com/dlang/druntime/blob/59607879d66de79e2fa159048ae74e52c7876ea6/src/core/sys/posix/dirent.d#L21-L28.

Reply:
@jacob-carlborg - That needs fixing in master too right?

Reply:
Yes, I didn't notice this PR targeted `dmd-cxx`.

Reply:
#2470.